legacy/src/main/java/org/opensearch/sql/legacy/query/planner/physical/node/scroll/Scroll.java

```diff
@@ -28,6 +28,7 @@
 import org.opensearch.sql.legacy.query.planner.physical.Row;
 import org.opensearch.sql.legacy.query.planner.physical.estimation.Cost;
 import org.opensearch.sql.legacy.query.planner.physical.node.BatchPhysicalOperator;
+import org.opensearch.sql.legacy.query.planner.physical.node.SearchHitRow;
```

Comment:
Yes this is because `SearchHitRow` was defined earlier at package level and did not require import.  Now  `PointInTime.java` class also utilizes the same class, it has been moved under `node` folder.